plugins/ec/meson.build

```diff
@@ -0,0 +1,31 @@
+if get_option('gudev')
+cargs = ['-DG_LOG_DOMAIN="FuPluginEc"']
+
+install_data(['ec.quirk'],
+  install_dir: join_paths(datadir, 'fwupd', 'quirks.d')
+)
+
+shared_module('fu_plugin_ec',
+  fu_hash,
+  sources : [
+    'fu-plugin-ec.c',
+    'fu-ec-device.c',
+    'fu-ec-it5570-device.c',
```

Comment:
Is this protocol specific to the it5570, or is it common to al the it55XX chips?

Reply:
@hughsie 
If you follow the discussion I linked above (https://review.coreboot.org/c/flashrom/+/55715) it may be common (at least some parts of it) to even more ITE chips (provided that they are using this class of firmware - described from the ITE EC framework). So it is more firmware framework-specific rather than a fully hardware feature.

```
Well, Tuxedo sells Clevo notebooks. Clevo uses ITE EC with ITE EC firmware framework. This code is *NOT* tuxedo-specific nor Clevo-specific, but ITE-specific and can be used with other ITE ECs as well (I know, because I know the original ITE tool - yes, a single one ;-) ). So, please do not artificially restrict this to Tuxedo. Thanks!
```

At that point we know for sure that the IT5570 needs different read/write functions (no idea if just 5570 or different 55xx chips as well). The other functions from `plugins/ec/fu-ec-it5570-device.c` can be **probably** shared between more devices.

Reply:
Okay, thanks. It looks common to other IT55xx to my eyes. I'll have a pull request for you in a couple of hours :)

Reply:
@macpijan can you have a look at wip/hughsie/superio please? If that works, please create a new pull request then we can review in detail. If it doesn't can you work out why, add a FIXUP commit to the branch and then do the PR thing please? I think using superio is a much nicer thing.

Reply:
@hughsie Thank you, @SergiiDmytruk will test and report.